std/algorithm/searching.d

```diff
@@ -2960,6 +3035,152 @@ unittest
     }
 }
 
+/**
+Iterates the passed range and returns the minimal element.
+A custom mapping for comparison key can be used with `map`.
+
+Params:
+    map = custom accessor for the comparison key
+    r = Range from which the minimal element will be selected
+
+Returns: The minimal element of the passed-in range.
+
+See_Also:
+    $(XREF min)
```

Comment:
Just wondering if this is going to link correctly as `min` is in another module?